Makefile

```diff
@@ -1,27 +1,33 @@
-
 PACKAGES=nonstd re re.posix re.glob cmdliner
 PACKAGES_INSTALL=$(PACKAGES)
```

Comment:
I think that you forgot to remove (or filter out) `re.posix` and `re.glob` here ?